### ISSUE TITLE
Write AssemblyInfo with UTF-8 and BOM

### ISSUE THREAD

**Issue by danquirk** [BMuuN/vsts-assemblyinfo-task#28]
I just enabled this task in my build and now my solution doesn't build because I am using the default StyleCop ruleset defined by the [Roslyn StyleCop Analyzers](https://github.com/DotNetAnalyzers/StyleCopAnalyzers)

```
src\include\GlobalAssemblyInfo.cs(1,1): Error SA1412: Store files as UTF-8 with byte order mark
```

Ideally this task would write the discovered AssemblyInfo files with the same format as they had previously. In my case that means UTF-8 and BOM, but it may also be a better default to use if the larger work to exactly match the existing format is too costly right now.

**Comment by javier-alvarez** [comment]
Same problem here

**Comment by BMuuN** [comment]
Your request has been implemented in v2.0.65. Apologies for the delay implementing this change!

Please note this upgrade is a breaking change and you should follow these steps to prevent any down-time:  https://github.com/BMuuN/vsts-assemblyinfo-task/wiki/Upgrade-v1-to-v2

This issue will now be closed.  If you feel the issue has not been fixed please feel free to re-open the issue.
